bigtop-packages/src/common/apex/do-component-build

```diff
@@ -28,6 +28,9 @@ command -v mvn > /dev/null 2>&1 || {
 # Build apex
 mvn clean package -DskipTests "$@"
 
+#Find hadoop.version from pom.xml
+APEX_HADOOP_VERSION=$(grep -oPm1 "(?<=<hadoop.version>)[^<]+" pom.xml)
```

Comment:
The very first material line of this file is to source `bigtop.bom` that contains the version of the components, included into the stack. Doing this grep you might effectively build Apex against a different version of Hadoop, hence introducing the inconsistency from the get go. 
Any reason you don't want to build this component against Bigtop's Hadoop version? Thanks.